apps/ima/translations/src/en/pages.json

```diff
@@ -54,26 +54,29 @@
     "header": "Declaration"
   },
   "immigration-adviser-declaration": {
+    "adviser": "Immigration adviser declaration",
     "header": "Declaration",
     "paragraph1": "What language was used to read the completed form to {{values.name}}?",
-    "paragraph2": "Include any dialects",
-    "header1": "Did you use an interpreter?"
+    "paragraph2": "Include any dialects"
   },
   "helper-declaration": {
+    "helper": "Helper declaration",
     "header": "Declaration",
     "paragraph1": "What language was used to read the completed form to {{values.name}}?",
-    "paragraph2": "Include any dialects",
-    "header1": "Did you use an interpreter?"
+    "paragraph2": "Include any dialects"
+  },
+  "summary": {
+    "summary": "Summary"
   },
```

Comment:
Not needed, I would remove summary.html and use confirm.html instead. summary.html will be used for a different step

Reply:
Removed